### PR TITLE
New check 7164 Cloudwatch log groups are protected by KMS @maisenhe

### PR DESCRIPTION
**Pull request by maisenhe**
Check if Cloudwatch log groups are protected by AWS KMS

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checks/check_extra7164 [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+CHECK_ID_extra7164="7.164"
+CHECK_TITLE_extra7164="[extra7164] Check if CloudWatch log groups are protected by AWS KMS "
+CHECK_SCORED_extra7164="NOT_SCORED"
+CHECK_TYPE_extra7164="EXTRA"
+CHECK_SEVERITY_extra7164="Medium"
+CHECK_ASFF_RESOURCE_TYPE_extra7164="Logs"
+CHECK_ALTERNATE_extra7164="extra7164"
+CHECK_SERVICENAME_extra7164="logs"
+CHECK_RISK_extra7164=""
```
Comment (toniblyx) [inline checks/check_extra7164]: Can you add here something like "Using customer managed KMS to encrypt CloudWatch log group provide additional confidentiality and control over the log data"

**Review by toniblyx — CHANGES_REQUESTED** [review]

File: checks/check_extra7164 (hunk 1 above)
Comment (maisenhe) [reply checks/check_extra7164]: I have updated the CHECK_RISK to the requested wording. Thanks

**Review by maisenhe — COMMENTED** [review]
I have made the requested changes and pushed it to the branch

**Review by toniblyx — APPROVED** [review]

**Comment by toniblyx** [comment]
Good job, thanks!

**Pull request merged**